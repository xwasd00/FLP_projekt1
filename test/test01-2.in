S,A,B,C,D
a,b,c,d
S
S->ABd
B->abd
A->BdC
A->BD
D->Cda
C->abc
