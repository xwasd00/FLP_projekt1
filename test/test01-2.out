S,A,B,C,D,<Bd>,<dC>,<bd>,<bc>,<da>,d',a',b',c'
a,b,c,d
S
S->A<Bd>
A->B<dC>
A->BD
B->a'<bd>
C->a'<bc>
D->C<da>
<Bd>->Bd'
<dC>->d'C
<bd>->b'd'
<bc>->b'c'
<da>->d'a'
d'->d
a'->a
b'->b
c'->c
